packages/block-library/src/button/edit.js

```diff
@@ -173,19 +165,18 @@ class ButtonEdit extends Component {
 					</PanelBody>
 				</InspectorControls>
 				{ isSelected && (
-					<URLPopover
-						focusOnMount={ false }
-						position={ popoverPosition }
-					>
-						<form
-							className="editor-format-toolbar__link-container-content"
-							onSubmit={ ( event ) => event.preventDefault() }>
+					<form
```

Comment:
I don't think the form here is needed anymore, it could be removed. The className could be applied to the BaseControl instead.

Reply:
Addressed [here](https://github.com/WordPress/gutenberg/pull/10128/commits/9ef9fdfc02d2da67a7a4be68245bb37bc6138d7d#diff-5ddfa233aa5293aedee4cdbb53926ef9). Thank you ❤️